pkg/converter/tool/builder.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package tool
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+var logger = logrus.WithField("module", "builder")
+
+type ConvertOption struct {
+	BuilderPath string
+
+	BootstrapPath    string
+	BlobPath         string
+	RafsVersion      string
+	SourcePath       string
+	ChunkDictPath    string
+	PrefetchPatterns string
+}
+
+type MergeOption struct {
+	BuilderPath string
+
+	SourceBootstrapPaths []string
+	TargetBootstrapPath  string
+	ChunkDictPath        string
+	PrefetchPatterns     string
+}
+
+func Convert(option ConvertOption) error {
+	args := []string{
+		"create",
+		"--log-level",
+		"warn",
+		"--prefetch-policy",
+		"fs",
+		"--bootstrap",
+		option.BootstrapPath,
+		"--blob",
+		option.BlobPath,
+		"--source-type",
+		"directory",
+		"--whiteout-spec",
+		"none",
+		"--fs-version",
+		option.RafsVersion,
+		"--blob-offset",
+		// Add blob offset for chunk info with size_of(tar_header) * 2.
+		"1024",
+	}
+	if option.RafsVersion != "" {
```

Comment:
args cannot work if `RafsVersion` is empty. Do you mean to disable check for rafs v6? Also IMO it is better to expose the checker option to users/callers instead of hardcoding it here.

Reply:
Fixed, added `RafsVersion` option to `ConvertOption`.